classAddress/classAddress.py

```diff
@@ -0,0 +1,118 @@
+import validation as valid
+
+class Address:
+    __postal_code_lenght = 5
+
+    def __init__(self, address_line = 'address_line', postal_code = 10000, country = 'country', city = 'city', fax_number = 100000, phone_number = 100000):
+        self.address_line = address_line
```

Comment:
use cycle here, method for getting property by name and dictionary (model of address)